test/test_ftp_object.jl

```diff
@@ -1,171 +1,325 @@
-using FTPClient
+@testset "FTPObject" begin
+
+    expected_header_port = r"229 Entering Extended Passive Mode \(\|\|\|\d*\|\)"
```

Comment:
I place all `expected_header_port` at the start of the files they are used in, so there's two of them. I could place it in runtests.jl, but I prefer to have the variables created and used in the same file for easy lookup. Thoughts.